container/Dockerfile.inf1-tf

```diff
@@ -0,0 +1,98 @@
+### Multi-stage Docker image. See https://docs.docker.com/develop/develop-images/multistage-build/
```

Comment:
I have some basic questions here. 
1. Are these containers supposed to use DLR to run Inferentia compiled models? Aren't we moving away from this?
    1. Is this why we are using MMS instead of TFS (like in DLC)? 
    2. Does this support the latest neuron compiler?